src/CoreImage/CIFilter.cs

```diff
@@ -199,6 +221,16 @@ internal int GetInt (string key)
 			}
 		}
 
+		internal nint GetNInt (string key)
+		{
+			using (var nskey = new NSString (key)){
+				var v = ValueForKey (nskey);
+				if (v is NSNumber)
+					return (v as NSNumber).NIntValue;
```

Comment:
Can it be done with just using the as? Like:

```csharp
var n = v as NSNumber;
return n?.NIntValue ?? 0;
```

Reply:
```csharp
if (v is NSNumber n)
    return n.NIntValue;
```

Reply:
it was a copy/paste of the existing `GetInt` method, but I can fix all of them
who's counting the lines anyway :)

Reply:
fixed in newer commits